bookwyrm/models/move.py

```diff
@@ -0,0 +1,50 @@
+""" move an object including migrating a user account """
+from django.db import models
+
+from bookwyrm import activitypub
+from .activitypub_mixin import ActivityMixin
+from .base_model import BookWyrmModel
+from . import fields
+from .status import Status
+
+
+class Move(ActivityMixin, BookWyrmModel):
+    """migrating an activitypub user account"""
+
+    user = fields.ForeignKey(
+        "User", on_delete=models.PROTECT, activitypub_field="actor"
+    )
+
+    # TODO: can we just use the abstract class here?
```

Comment:
I don't think you can set a field to reference an abstract class, unfortunately. Since I didn't design a good Actor base model and haven't managed that large a refactor, I'm not sure how to handle arbitrary types besides subclassing the Move object for different object types?